.github/workflows/rust.yml

```diff
@@ -25,6 +25,20 @@ jobs:
     - run: rustup update
     - run: cargo fmt --all --check
 
+  cargo-deny:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        checks:
+          - advisories bans licenses sources
+    # Prevent sudden announcement of a new advisory from failing ci:
+    continue-on-error: ${{ matrix.checks == 'advisories' }}
```

Comment:
Is the goal to run this job four times in parallel, one time for each of the words listed here? If so, they need to be an array. Right now it is running one job with the `checks` being one value, this long value.

```suggestion
    strategy:
      matrix:
        checks: ["advisories", "bans", "licenses", "sources"]
    # Prevent sudden announcement of a new advisory from failing ci:
    continue-on-error: ${{ matrix.checks == 'advisories' }}
```

```suggestion
    strategy:
      matrix:
        checks:
        - advisories
        - bans
        - licenses
        - sources
    # Prevent sudden announcement of a new advisory from failing ci:
    continue-on-error: ${{ matrix.checks == 'advisories' }}
```

Reply:
Oh, huh. So .. the example I copied this from had `advisories` in one matrix entry, and `bans licenses sources` in another. And I think the `continue-on-error` line there is designed with that organization in mind. I'll set it back to that (it works in all cases -- you can pass multiple checks on the same invocation -- but the idea with the 2-entry organization is to allow the `advisory` version not to block CI since it happens asynchronously, outside the user's control)